frontend/components/Filter/FilterBanner.tsx

```diff
@@ -9,8 +9,19 @@ import TennisImage from "./../../public/images/tennis-balls.png";
 import VolleyballImage from "./../../public/images/volleyball.png";
 import FitlerDialog from "./FilterDialog";
 import FilterIcon from "./FilterIcon";
+import { EventData } from "@/interfaces/EventTypes";
 
-export default function FilterBanner() {
+type FilterBannerProps = {
```

Comment:
im not picky, but can we decide on a props structure and naming convention. Currently eventhing is `I + name of class`, but im happy to convert it to `Name of class + props`..

also advise me on why you use `type` instead of interface, Im not too good at js :(

Reply:
I think convention is to use **interface** with Name of class + props
https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/typescript-best-practices.html

I'll go ahead and change that throughout the code base on my branch

Reply:
For props, we'll do name of class + Props. For normal interface, we'll do I + name of class